.github/workflows/actions/prepare-binaries/action.yml

```diff
@@ -23,6 +23,6 @@ runs:
         chmod u+x ./tmp/polkadot* ./tmp/staking-miner-playground
         ./tmp/polkadot --version
         ./tmp/staking-miner-playground --version
-        mkdir -p ~/.local/bin
-        mv ./tmp/polkadot* ~/.local/bin
-        mv ./tmp/staking-miner-playground ~/.local/bin
+        mkdir -p /usr/local/bin
```

Comment:
I changed this path because it was not found in the default path, not sure how it worked before :thinking: